stats.go

```diff
@@ -266,12 +278,17 @@ func (c *gauge) Value() uint64 {
 }
 
 type timer struct {
+	base time.Duration
 	name string
 	sink Sink
 }
 
 func (t *timer) time(dur time.Duration) {
-	t.sink.FlushTimer(t.name, float64(dur/time.Microsecond))
+	t.AddDuration(dur)
+}
+
+func (t *timer) AddDuration(dur time.Duration) {
+	t.sink.FlushTimer(t.name, float64(dur/t.base))
```

Comment:
this is maybe a "i'm not a go programmer" piece of feedback, but the `float64(dur/t.base)` doesn't make a lot of sense unless you happen to know that:

- durations in go are actually nanoseconds
- `time.Millisecond` and `time.Microsecond` are the amount of nanoseconds in a ms or a µs

thoughts on adding a comment explicating what this does?